fs/fsx.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+# Copyright: 2017 IBM
+# Author: Harish <harish@linux.vnet.ibm.com>
+#
+
+import os
+from avocado import Test
+from avocado import main
+from avocado.utils import process
+from avocado.utils.software_manager import SoftwareManager
+
+
+class Fsx(Test):
+    '''
+    The Fsx test is a file system exerciser test
+    '''
+
+    def setUp(self):
+        '''
+        Setup fsx
+        '''
+        smm = SoftwareManager()
+        deps = ['gcc', 'make']
+        for package in deps:
```

Comment:
can this be like for package in ['gcc', 'make'] just want to eliminate extra variable